pkg/datapath/iptables/iptables.go

```diff
@@ -101,6 +101,24 @@ func getFeedRule(name, args string) []string {
 	return append(argsList, ruleTail...)
 }
 
+// KernelHasNetfilter probes whether iptables related modules are present in
+// the kernel and returns true if indeed the case, else false.
+func KernelHasNetfilter() bool {
+	modulesManager := &modules.ModulesManager{}
+	if err := modulesManager.Init(); err != nil {
+		return true
+	}
+	if found, _ := modulesManager.FindModules(
```

Comment:
what about the running this function if either the `option.Config.EnableIPv4` or `option.Config.EnableIPv6` are enabled?

Reply:
(Discussed offline wrt above answer. Ok to proceed.)